wolt/venue.go

```diff
@@ -67,6 +70,11 @@ func ParseVenue(venuesJSON []byte) (*Venue, error) {
 		return nil, fmt.Errorf("venue coordinate from array: %w", err)
 	}
 
+	v.TimezoneLocation, err = time.LoadLocation(v.Timezone)
```

Comment:
I would check here if timezone is empty, put the default timezone (maybe sometimes Wolt will return an empty timezone)

Reply:
Which one is the default timezone? Currently, in case Wolt returns an empty timezone then `TimezoneLocation` is equal to UTC, because of `time.LoadLocation`'s defaults.

Perhaps the best solution is to use Slack's [date formatting](https://api.slack.com/reference/surfaces/formatting#date-formatting), which displays times in the recipient's timezone.
Slack's format receives a fallback, which I've set to the venue's timezone.
If that timezone is missing (which I hope won't happen), then it falls back to UTC, as explained above.

Reply:
In that case it seems fine to me